packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -77,20 +81,18 @@ export function Drawer({
   // Adds body class while open so the toolbar nav can hide some actions while drawer is open
   useBodyClassWhileOpen();
 
-  // Apply size styles (unless deprecated width prop is used)
-  const rootClass = cx(styles.drawer, !width && styles.sizes[size]);
+  const rootClass = cx(styles.drawer, sizeStyles);
   const content = <div className={styles.content}>{children}</div>;
 
   return (
     <RcDrawer
       open={true}
       onClose={onClose}
       placement="right"
-      // Important to set this to empty string so that the width can be controlled by the css
-      width={width ?? ''}
       getContainer={'.main-view'}
       className={styles.drawerContent}
       rootClassName={rootClass}
+      width={''}
```

Comment:
might want to leave the comment on here (or, try and remove it) - this looks like a smell and i would be tempted to remove it if i didn't know any better

Reply:
@joshhunt oh, yes. good point! will add it back